change/@fluentui-react-charting-20c8ae70-f2a4-4ef4-89b6-7ca932f7481d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
The issue is reproing for stacked bar chart as well.

Reply:
fixed in stacked bar chart also